src/screens/PopUps/UpdateRequired/index.tsx

```
@@ -19,63 +21,25 @@ export const UpdateRequired = () => {
     return () => backHandler.remove();
   }, []);
 
+  const onUpdatePress = async () => {
+    try {
+      if (isIOS) {
+        await Linking.openURL(APP_STORE_LINK);
+      } else {
+        await Linking.openURL(PLAY_STORE_LINK);
+      }
+    } catch {}
```

Use `logError`, never leave empty catch blocks

---

logError added